feat(SR): Add FAQ note about SR on non-iOS (#13602)
getsentry/sentry-docs: docs/platforms/apple/guides/ios/session-replay/index.mdx

Q: **Why is my issue missing a replay?**

A: An issue may be missing a replay because the user's device was [offline](/product/explore/session-replay/mobile#frequently-asked-questions) while `sessionSampleRate` was specified, your project/organization was rate-limited, or (in rare cases) the device failed to capture the replay video.

Q: **Does Session Replay work on tvOS and visionOS?**

> A: We don`t actively prevent you from using Session Replay on these platforms, but currently we only officially support it on iOS and cannot make guarantees about the functionality and the performance on other platforms.

Reviewer: ```suggestion
A: We don't actively prevent you from using Session Replay on these platforms, but we only officially support it on iOS. As a consequence, we can't make guarantees about its functionality and performance on other platforms.
```
